### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SKulenko**
Updated readme.

### CODE REVIEW

**Comment by SKulenko** [comment]
recheckcla

File: README.md [hunk 1]
```diff
@@ -1,423 +1,394 @@
...
-We use one of the most popular face recognition methods based on [deep neural networks](#used-ml-papers-and-algorithms), and provide a convenient API for Face Collection training and face recognition. We also provide a convenient roles system with which you can easily control who has access to the Face Collection.
+To get started:
 
-Every user can create several Face Collections trained on different subsets of people. 
+1. Install Docker
+2. Download the archive from our latest release: [https://github.com/exadel-inc/CompreFace/releases](https://github.com/exadel-inc/CompreFace/releases)
+3. Unzip the archive
+4. Run Docker
+5. Windows search bar-\&gt; cmd-\&gt;in the Command prompt-\&gt; cd -\&gt;paste the path to the extracted zip folder
+6. Run command: docker-compose up --build
```
Comment (pospielov) [inline README.md:33]: for usual users it's just docker-compose up
```suggestion
6. Run command: docker-compose up
```

**Comment by pospielov** [comment]
Please review readme again:
https://github.com/SKulenko/CompreFace/tree/patch-1
image
It looks like some problems with Markdown - there are some symbols (&lt;), something is in the table, but shouldn't, and remove please retrain_option everywhere

**Comment by pospielov** [comment]
Also, please update architecture photo

**Pull request closed without merging**